src/main/resources/add-ons-to-index.json

```diff
@@ -1421,6 +1439,7 @@
       "uid": "org.openmrs.module.htmlwidgets",
       "type": "OMOD",
       "name": "HTML Widgets",
+      "icon": "html5",
```

Comment:
Not everyone will recognize the logo. Hence, it might seem as though this icon is not matching the module name and hence I feel this must be deleted